tests/framework-tests/preferences/js/PanelTestUtils.js

```diff
@@ -42,4 +45,10 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         jqUnit.assertEquals(message, expectedState, checkbox.is(":checked"));
     };
 
+    fluid.tests.panels.utils.getTemplate = function (that) {
```

Comment:
Maybe I should have said grade instead of component, but what I meant was that it should probably be called "fluid.tests.panels.utils.injectTemplates.getTemplate".